app/js/app/directives/react_classes/UnknownBlock.js

```diff
@@ -36,17 +36,19 @@ define(["react", "jquery"], function(React,$) {
 			render: function() {
 				return (
 					<Block type="unknown" blockTypeTitle="?" doc={this.props.doc} onChange={this.onDocChange}>
-						<div className="row">
+						<div className="row">{this.props.doc}
```

Comment:
This shows horribly on the 'select content block type' menu, I don't think it should be here?

Reply:
Indeed it should not, it was debugging stuff and I must have forgotten to remove it.